src/Controller/RegisterController.php

```diff
@@ -57,7 +57,7 @@ public function createUser(SerializerInterface $serializer, Request $request, Us
         $data = $request->getContent();
 
         /** @var DeserializationContext $context */
-        $context = DeserializationContext::create();
+        $context = DeserializationContext::create()->setGroups(array('UserCreate'));
 
         $userDTO = $serializer->deserialize(
```

Comment:
Inject the serializer in `__construct()` method like `validator` and `transformer`